Salt package lock #15572 keichwa

keichwa opened the pull request:
# Description

https://github.com/SUSE/spacewalk/issues/15572

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

mcalmer commented on modules/client-configuration/pages/registration-bootstrap.adoc:
```diff
@@ -157,12 +157,15 @@ To review a package EULA, open the package detail page in the {webui}.
-The feature is not currently supported on {rhel} or Salt clients.
+Package locks can be used on traditional and Salt clients that use the Zypper package manager.
+Package locks can also be used on some other Salt clients.
+On {rhel}-like systems (version 7), you must have the [package]``yum-plugin-versionlock`` package installed.
```
On RHEL 8 locking is supported as well. The plugin package is named a bit different:

  python3-dnf-plugin-versionlock

mcalmer replied:
And Debian/Ubuntu is supported as well. `apt` has the support included.

mcalmer reviewed (changes requested):
Btw: There is one big difference between package lock with zypp and the others.
With zypp you can also lock packages which are not installed. This prevent installing the the package. This is not possible for RHEL or Debian like systems. They can only lock packages which are installed.
Our UI reflect this, we we show only installed packages for RHEL and Debian like systems while for SUSE systems we show all available packages.

keichwa commented:
> Btw: There is one big difference between package lock with zypp and the others.
> With zypp you can also lock packages which are not installed. This prevent installing the the package. This is not possible for RHEL or Debian like systems. They can only lock packages which are installed.
> Our UI reflect this, we we show only installed packages for RHEL and Debian like systems while for SUSE systems we show all available packages.

Thanks for input, @mcalmer --I hope I addressed all your concerns.  Can you please review again?

mcalmer reviewed (approved):
LGTM

vzhestkov reviewed (approved):
Looks good

keichwa commented:
Thanks for your approvals, Michael and Victor!  Now, @0rnela / @jcayouette I need at least one tic of my co-editors, too ;)
